packages/fx-core/resource/unify-config-change-logs.md

```diff
@@ -2,6 +2,13 @@
 
 We have updated the configuration files so that your project is compatible with the latest Teams Toolkit features, including a consistent configuration file schema and a single Teams app manifest template across local and remote environments.
 
+> Important Notes: If you collaborate on this project with your co-workers, please ensure your team members update the Teams Toolkit extension to the latest version after committing the changes with this upgrade.
```

Comment:
@MuyangAmigo can you help to review this?

Reply:
This is what I have updated. Do you have any particular comments?

Reply:
I saw it was different from what in the wiki page, so was wonder which one we should use. 
Now I see wiki page is also updated with this change, so no more question here. Thanks @MuyangAmigo and @xiaolang124